modules/mobile/app/controllers/mobile/v0/recipients_controller.rb

```diff
@@ -19,8 +19,11 @@ def all_recipients
         raise Common::Exceptions::ResourceNotFound if resource.blank?
 
         resource.records = resource.records.reject(&:blocked_status)
+        resource.records = resource.records.select(&:preferred_team)
```

Comment:
This is also being checked but the FE of VAHB, but I wanted to add it here so that the care systems list would only include systems with at least 1 preferred team. When I update the FE I will remove the check and just let the backend handle it